planning_period: 0.01 # seconds
world_frame: "local_origin_stable"
min_balloon_height: 0.5 # meters
gating_distance: 4.0 # meters
max_time_since_update: 2.0 # seconds
min_updates_to_confirm: 10

process_std:
  position: 1.0
  yaw: 1.0
  speed: 1.0
  curvature: 1.0
  quaternion: 1.0

init_std:
  yaw: 1.0
  speed: 1.0
  curvature: 1.0
  quaternion: 1.0
